README.md

```diff
@@ -15,78 +17,221 @@
 - from [GitHub\*](https://github.com/oneapi-src/oneDAL/releases).
 
+### Installation from Source
 See [Installation from Sources](INSTALL.md) for details.
 
 ## Examples
 
-Examples for different programming languages:
-
-- [C++](https://github.com/oneapi-src/oneDAL/tree/master/examples/cpp)
-- [C++ SYCL*](https://github.com/oneapi-src/oneDAL/tree/master/examples/cpp_sycl)
-- [Java](https://github.com/oneapi-src/oneDAL/tree/master/examples/java)
-
-Data Examples for different computation modes:
-
-- [Batch](https://github.com/oneapi-src/oneDAL/tree/master/examples/data/batch)
-- [Distributed](https://github.com/oneapi-src/oneDAL/tree/master/examples/data/distributed)
-- [Online](https://github.com/oneapi-src/oneDAL/tree/master/examples/data/online)
+Except C++ and Python API oneDAL also provide API for C++ SYCL and Java languages. Check out tabs below for more examples.
+- [C++](https://github.com/oneapi-src/oneDAL/tree/master/examples/daal/cpp)
+- [oneAPI C++](https://github.com/oneapi-src/oneDAL/tree/master/examples/oneapi/cpp)
+- [oneAPI DPC++](https://github.com/oneapi-src/oneDAL/tree/master/examples/oneapi/dpc)
+- [Java](https://github.com/oneapi-src/oneDAL/tree/master/examples/daal/java)
+- [Python](https://github.com/IntelPython/daal4py/tree/master/examples)
 
 ## Documentation
-
 - [Get Started](http://oneapi-src.github.io/oneDAL/getstarted.html)
-- [System Requirements](https://software.intel.com/en-us/articles/system-requirements-for-oneapi-data-analytics-library#)
+- [System Requirements](https://software.intel.com/content/www/us/en/develop/articles/system-requirements-for-oneapi-data-analytics-library.html)
 - [oneDAL documentation](http://oneapi-src.github.io/oneDAL/)
-- [Specifications](https://spec.oneapi.com/versions/latest/elements/oneDAL/source/index.html)
-- [Release Notes](https://software.intel.com/en-us/articles/oneapi-dal-release-notes)
+- [Specification](https://spec.oneapi.com/versions/latest/elements/oneDAL/source/index.html)
+- [Release Notes](https://software.intel.com/content/www/us/en/develop/articles/oneapi-dal-release-notes.html)
 - [Known Issues](https://oneapi-src.github.io/oneDAL/notes/known_issues.html)
 
-## API
+## Samples
+Samples is an examples of how oneDAL can be used in different applications.
+- [Apache Arrow](https://github.com/oneapi-src/oneDAL/tree/master/samples/daal/cpp/arrow)
+- [KDB](https://github.com/oneapi-src/oneDAL/tree/master/samples/daal/cpp/kdb)
+- [MPI](https://github.com/oneapi-src/oneDAL/tree/master/samples/daal/cpp/mpi)
+- [MySQL](https://github.com/oneapi-src/oneDAL/tree/master/samples/daal/cpp/mysql)
+- [oneCCL](https://github.com/oneapi-src/oneDAL/tree/master/samples/daal/cpp/oneccl)
+- [Hadoop](https://github.com/oneapi-src/oneDAL/tree/master/samples/daal/java/hadoop)
+- [Java Spark](https://github.com/oneapi-src/oneDAL/tree/master/samples/daal/java/spark)
+- [Scala Spark](https://github.com/oneapi-src/oneDAL/tree/master/samples/daal/scala/spark)
```

Comment:
I think we need to extend description of the samples. **But we can to it in separate PR.**